### PR TITLE
fix(tests): fix tests to run correctly

### PR DESCRIPTION
**Pull request by dimaip**
In future client tests would go to `test-client` command, so we need ot
prepare for that.

Closes: #21

### CODE REVIEW

**Comment by dimaip** [comment]
:+1:

File: package.json [hunk 1]
```diff
@@ -4,7 +4,9 @@
   "description": "chat",
   "main": "index.js",
   "scripts": {
-    "test": "./node_modules/.bin/mocha --compilers js:babel/register --require ./server/test/test_helper.js  --recursive",
+    "test-server": "./node_modules/.bin/mocha --compilers js:babel/register --require ./server/test/test_helper.js --recursive ./server/test/test_core.js",
+    "test-server:watch": "npm run test-server -- --watch",
+    "test": "npm run test-server",
     "test:watch": "npm run test -- --watch",
```
Comment (Nitive) [inline package.json]: this command doesn't work

**Comment by dimaip** [comment]
:+1:

**Comment by Nitive** [comment]
++

**Comment by Nitive** [comment]
Сливай

**Pull request merged**